content/en/docs/tasks/administer-cluster/manage-resources/quota-memory-cpu-namespace.md

```diff
@@ -141,6 +141,17 @@ Error from server (Forbidden): error when creating "examples/admin/resource/quot
 pods "quota-mem-cpu-demo-2" is forbidden: exceeded quota: mem-cpu-demo,
 requested: requests.memory=700Mi,used: requests.memory=600Mi, limited: requests.memory=1Gi
 ```
+{{< note >}}
+This is an exception for enforcing resource quota on ephemeral-storage.
+
+To make the resource quota work on ephemeral-storage, two things need to be done.
+
+1. an admin sets the resource quota for ephemeral-storage
+2. User needs to mention the ephemeral-storage resource limit in the Pod spec.
```

Comment:
Do NOT use explicit numbers for ordered list, unless you have a strong reason to do so.

```suggestion
1. User needs to mention the ephemeral-storage resource limit in the Pod spec.
```

Reply:
Done